android/src/main/kotlin/com/posthog/posthog_flutter/PosthogFlutterPlugin.kt

```diff
@@ -15,37 +15,29 @@ import io.flutter.plugin.common.MethodChannel.Result
 
 /** PosthogFlutterPlugin */
 class PosthogFlutterPlugin : FlutterPlugin, MethodCallHandler {
-    /// The MethodChannel that will the communication between Flutter and native Android
```

Comment:
Some comments were removed, please add them back, those comments are good for new comers